verify.go

```diff
@@ -26,6 +26,12 @@ func verify(uri string, cache DiscoveryCache, getter httpGetter, nonceStore Nonc
 	// When the Relying Party receives a positive assertion, it MUST
 	// verify the following before accepting the assertion:
 
+	// - The value of "openid.signed" contains all the required fields.
```

Comment:
Those strings were mostly copy-pasted from the spec. The spec is sort of implicit by saying "when the relying party receives a positive assertion, bla bla", implying the RP must check that this is a positive assertion (which you just added).
Could you make the comment start with "First we verify that this is a positive assertion (Section 10.1) ? I just want to make it clear what comes from the spec (bullet points) and what doesn't.